frontend-react/src/network/api/WatersApi.ts

```diff
@@ -30,6 +30,7 @@ export interface ResponseError {
     scope: string | undefined;
     trackingIds: string[] | undefined;
     details: any | undefined;
+    rowList?: string;
```

Comment:
instead of including this in the ResponseError type, make a new type to map onto